index.js

```diff
@@ -12,7 +14,7 @@
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
 // console.log("%cQuestion 2", "color: red") 
-
+const name = "Andre"
```

Comment:
Good! The key here was using `const` because it doesn't allow us to reassign our variable